src/main/java/DukeException.java

```diff
@@ -0,0 +1,7 @@
+public class DukeException extends Throwable {
```

Comment:
the websites says: "Class variables should never be declared public"
So, perhaps it should not public.